elixir/apps/web/lib/web/live/settings/identity_providers/openid_connect/components.ex

```diff
@@ -6,14 +6,21 @@ defmodule Web.Settings.IdentityProviders.OpenIDConnect.Components do
     <div class="max-w-2xl px-4 py-8 mx-auto lg:py-12">
       <.form for={@form} phx-change={:change} phx-submit={:submit}>
         <.step>
-          <:title>Step 1. Create OAuth app in your identity provider</:title>
+          <:title>Step 1. Create OAuth client in your identity provider</:title>
```

Comment:
Just curious on the change here.  I've always seen IDPs use the 'app' terminology, such as: [Okta Docs](https://help.okta.com/en-us/content/topics/apps/apps_app_integration_wizard_oidc.htm)

Just wondering if you had seen different.

Reply:
Yeah, we could use `app`, I think I've seen a similar naming pattern. Was just trying to be consistent. I'll update these.

Reply:
Fixed